draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1684,9 +1684,20 @@ and v2 SCTs to co-exist in a certificate (See {{v1_coexistence}}).
 In addition to normal validation of the server certificate and its chain, TLS
 clients SHOULD validate each received SCT for which they have the corresponding
 log's parameters. To validate an SCT, a TLS client computes the signature input
-from the SCT data and the server certificate, and then verifies the signature
-using the corresponding log's public key. TLS clients MUST NOT consider valid
-any SCT whose timestamp is in the future.
+by constructing a `TransItem` of type `x509_entry_v2` or `precert_entry_v2`,
+depending on the SCT's `TransItem` type. The `TimestampedCertificateEntryDataV2`
+structure is constructed in the following manner:
+
+* `timestamp` is copied from the SCT.
+* `tbs_certificate` is the TBSCertificate portion of the server certificate.
+* `issuer_key_hash` is extracted from the certificate in the chain that
+  certified the server certificate.
+* `sct_extensions` is copied from the SCT.
+
+The signature (from the SCT's `signature` field) is then verified using the
+corresponding log's public key (identified by the `log_id` field).
```

Comment:
"log_id" doesn't identify the log's public key, per se.  Rather, it identifies the log.
Also, we should mention that the signature algorithm is one of the log's parameters.

Something like this:
`The SCT's "signature" is then verified using the public key of the corresponding log, which is identified by the "log_id".  The required signature algorithm is one of the log's parameters."`